coralproject/talk
# do not use hjson for plugins.env.js

## What does this PR do?
fixes a crush when using TALK_PLUGINS_JSON introduced in https://github.com/coralproject/talk/commit/36e855237fc0f5509f8271dbe6ed97e5ca0180af

```
talk:plugins Now using TALK_PLUGINS_JSON environment variable for plugins +0ms
coral/plugins.js:204
const iteratePlugins = plugins => plugins.map(p => new Plugin(p));
                                          ^
TypeError: plugins.map is not a function
    at iteratePlugins (~/coral/plugins.js:204:43)
```

## How do I test this PR?
- set TALK_PLUGINS_JSON 
- run talk
- verify that plugins from TALK_PLUGINS_JSON are loaded
